src/shared/ui/link-custom/link-custom.tsx

```diff
@@ -1,42 +1,56 @@
 /* eslint-disable @stylistic/jsx-one-expression-per-line */
 import { AnchorHTMLAttributes, DetailedHTMLProps } from 'react';
+import { type VariantProps, cva } from 'class-variance-authority';
 import classNames from 'classnames/bind';
 import { Link } from 'react-router-dom';
 
 import styles from './link-custom.module.scss';
 
+export const cx = classNames.bind(styles);
+
 type LinkCustomProps = DetailedHTMLProps<
   AnchorHTMLAttributes<HTMLAnchorElement>,
   HTMLAnchorElement
-> & {
+> &
+VariantProps<typeof linkCustomVariants> &
+LinkCustomAdditionalProps;
```

Comment:
```tsx
type LinkCustomProps = HTMLAttributes<HTMLAnchorElement> &
  VariantProps<typeof linkCustomVariants> &
  LinkCustomAdditionalProps;
  ```

Reply:
Throw and error:
Property 'target' does not exist on type 'IntrinsicAttributes & HTMLAttributes<HTMLAnchorElement>

Reply:
My bad, this is the correct one

```tsx
type LinkCustomProps = AnchorHTMLAttributes<HTMLAnchorElement> &
  VariantProps<typeof linkCustomVariants> &
  LinkCustomAdditionalProps;
```

Reply:
Fixed